### PR TITLE
Update to Ubuntu 20.04

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Switch remaining ubuntu 18 vms to 20

### CODE REVIEW

**Review by weshaggard — APPROVED** [review]
Changes look good once we get some test validation on them.

**Review by weidongxu-microsoft — APPROVED** [review]

File: eng/pipelines/templates/stages/archetype-sdk-management.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ stages:
           JavaVersion: '1.8'
 
     pool:
-      vmImage: 'ubuntu-18.04'
```
Comment (weshaggard) [inline eng/pipelines/templates/stages/archetype-sdk-management.yml:26]: @weidongxu-microsoft are you guys good with moving your pipelines to ubuntu-20.04? 

Also we should chat about moving your pipelines to use our new 1ES Agent pools (see other changes in this PR with the name for those).

Comment (weidongxu-microsoft) [reply eng/pipelines/templates/stages/archetype-sdk-management.yml]: Sure. I assume it going to follow the configure as `archtype-sdk-client.yml`?
```
- name: MatrixConfigs
  type: object
  default:
    - Name: Java_ci_test_base
      Path: eng/pipelines/templates/stages/platform-matrix.json
      Selection: sparse
      NonSparseParameters: Agent
      GenerateVMJobs: true
```

On the other hand, this template is used for track1 mgmt SDK. We haven't updated or released them for a couple of months. Though I guess for the sake of code reference, we still need to keep the code in main, but maybe we can schedule to skip the CI on them.

**Pull request merged**